src/modules/display.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable import/no-cycle */
+
+import data from './data.js';
+import { remove, changeValue, getFromLocal } from './addRemove.js';
+import checkBoxValue from './checkbox.js';
+
+const displayList = () => {
+  getFromLocal();
+  const listUl = document.querySelector('.list-container');
+  listUl.innerHTML = null;
+  let listText;
+  for (let i = 0; i < data.toDoArr.length; i += 1) {
+    const { description } = data.toDoArr[i];
+    const { index } = data.toDoArr[i];
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.name = 'task-completed';
+    checkbox.value = 'false';
+    checkbox.id = `${i + 1}`;
+    const listElement = document.createElement('li');
+    const liWraper = document.createElement('div');
+    const textList = document.createElement('input');
+    textList.type = 'text';
+    textList.name = 'toDoElement';
+    const iconDots = document.createElement('span');
+    liWraper.className = 'li-wraper';
+    listElement.className = 'li-element';
+    textList.className = `input-list-element ${(index)}`;
+    listText = description;
+    textList.value = listText;
+    iconDots.innerHTML = '<i class="fa-solid fa-ellipsis-vertical"></i>';
+    iconDots.setAttribute('id', i + 1);
+    liWraper.append(checkbox);
+    liWraper.append(textList);
+    liWraper.append(iconDots);
+    listElement.append(liWraper);
+    listUl.append(listElement);
```

Comment:
Nice job so far !!
DOM manipulation is a very expensive task so please do use fragment's as much as possible or use strings to create elements on the fly. here is a doc to refer https://developer.mozilla.org/en-US/docs/Web/API/DocumentFragment

Reply:
ok ok thanks  for the advice